### DISCUSSION TITLE
Add support for sending the output of commands as messages

### DISCUSSION THREAD

**Discussion by syswraith** [d99kris/nchat#285] [Ideas]
Feature request for adding vi(m)-like shell integration for sending the output of shell commands as messages directly as messages. Similar to how vi implements line and section wands (more info about it at  https://rwx.gg/tools/editors/vi/how/magic/)

**Comment by ferroflux** [comment]
Interesting feature. Might make sending code over the terminal easier. Upvoting this.

**Comment by lcarcamo1526** [comment]
This tool is exactly what I was looking for. It would be pretty cool to use a Vim navigation style!
